radio/src/maths.cpp

```diff
@@ -172,3 +173,30 @@ void getGpsDistance()
     telemetryData.hub.maxGpsDistance = telemetryData.hub.gpsDistance;
 }
 #endif
+
+#if defined(STM32F4)
+uint32_t getCoordDistance(float  y1, float x1, float y2, float x2){
+  float nRadius = 6370997.0; // Earth’s radius in meters
```

Comment:
How was this radius value arrived at?  Seems close to "mean radius" but not exactly...  Anyway a fixed radius like this will limit precision unless you happen to be exactly at the matching latitude.  

Also, when using floats, always add an `f` at the end of the number.  This should be default behavior (i.e. good habit) unless you specifically need a double.